src/Kunstmaan/TranslatorBundle/Resources/config/services.yml

```diff
@@ -109,6 +109,7 @@ services:
 
     kunstmaan_translator.service.command.diff:
         class: 'Kunstmaan\TranslatorBundle\Service\Command\DiffCommand'
+        deprecated: 'The "%service_id%" service is deprecated since TranslatorBundle 5.2 and will be removed in TranslatorBundle 6.0.'
```

Comment:
Mention "KunstmaanTranslatorBundle" instead of "TranslatorBundle" because these deprecations will show up for the whole application, this way the user knows which bundle causes the deprecation

Reply:
True, will change the name.